storage/s3.go

```diff
@@ -128,7 +128,7 @@ func (s *S3) Get(ctx context.Context, path string) (io.ReadCloser, error) {
 	}
 	out, err := s.s3client.GetObject(ctx, input)
 	if err != nil {
-		return nil, errors.Wrapf(err, "fail to get object %v", path)
+		return nil, errors.Wrapf(err, "fail to get S3 object %v", path)
```

Comment:
I found it unclear in the Rollbar errors that the storage backend is not in the error message. Hence I wanted to add it. It's a separated commit so if you don't like it I can revert easily.